dessia_common/workflow/blocks.py

```diff
@@ -187,7 +187,7 @@ def dict_to_object(cls, dict_: JsonSerializable, force_generic: bool = False,
         class_ = get_python_class_from_class_name(classname)
         name = dict_['name']
         method_type = ClassMethodType(class_=class_, name=method_name)
-        return cls(method_type=method_type, name=name)
+        return cls(method_type=method_type, name=name, position=dict_.get('position'))
```

Comment:
Why not just `position=dict_["position"]` ? Seems more readable to me, as we do not define any default value to the get.

This goes for all the others, as well

Reply:
`dict_['position']` raises an error if position is not in dict_
Whereas dict_.get('position') returns None in such a case.

The point of this dev was to call the init with None if no explicit position has been set. 
In this case the init changes the None value for (0,0)
-> This answer another of your comment where you say that I can directly write 
```py
    def __init__(self, inputs: List[Variable], outputs: List[Variable],
                 position: Tuple[float, float] = (0,0), name: str = ''):
```

Reply:
Alright, I thought one was supposed to write `dict_.get("stuff", default_value)` to have a default value, even for None.
That seems all good to me, then !